content/applications/general/product_images.rst

```diff
@@ -0,0 +1,133 @@
+   to generate Google Custom Search API credentials. Log in with your Google account.
+
+#. Select or create an API project to store the credentials if not yet done before. Give it an 
+   explicit name (e.g. Odoo Images).
+
+#. In the credentials section, click on `Create Crendentials` and select `API Keys`.
+
+   .. image:: product_images/google-images-credentials00.png
+      :align: center
+
+#. Save it, you'll need it for the next step in Odoo !
+
+#. Use the search bar to find for `Google Custom Search API` and select it.
+
+   .. image:: product_images/google-images-credentials01.png
+      :align: center
+
+#. Enable the API.
+
+   .. image:: product_images/google-images-credentials02.png
+      :align: center
+
+.. _product_images/google-pse-dashboard:
+
+Google Programmable Search dashboard
+------------------------------------
+
+#. Go to `Google Programmable Search Engine <https://programmablesearchengine.google.com/>`__ and
+   click on `Get Started`. Log in with your Google account.
+
+   .. image:: product_images/google-images-credentials03.png
+      :align: center
+
+#. Fill the language and the name of the search engine. Give it an explicit name
+   (e.g. Odoo Images).
+
+   .. note::
+      Google doesn't allow to create a search engine without having entered at least one specific
+      site to search on. You can put any website (e.g. www.google.com) for this step, we will
+      remove it later.
+
+#. Validate the form by clicking on `Create`. Then, go to the edition mode of the search engine
+   that you created (either by clicking on `Control Panel` on the confirmation page or by
+   clicking on the name of your Search Engine on the Home page).
+
+#. In the basics tab, make sure to enable Image search, SafeSearch and Search the entire web.
+
+   .. note::
+      Once `Search the entire web` enabled, you can safely delete the site that you previously
+      entered at the previous step.
+
+#. Save your **Search Engine Id**.
+
+   .. tip::
+      You can easily save your Search Engine ID by clicking on the Copy to clipboard button next to
+      it.
+
+.. _product_images/setup-in-odoo:
+
+Odoo
+----
+
+#. Go to :menuselection:`Settings --> General Settings --> Integrations`,
+   activate **Google Images** and save.
+
+#. Go back to :menuselection:`Settings --> General Settings`, enter your **API Key** and
+   **Search Engine ID** in Google Images settings and save again.
+
+#. The setup is now ready.
+
+.. _product_images/get-product-images:
+
+Automatically get your product images in Odoo
+=============================================
+
+The action to automatically get your product images in Odoo appears in any Products or Product
+Variants list view. Here is a step-by-step guide from the Inventory app.
+
+#. Go to :menuselection:`Inventory --> Products --> Products` or :menuselection:`Inventory -->
+   Products --> Product Variants`.
```

Comment:
Maybe too specific, could be useful to explain that any product menu does the trick.

Reply:
What do you think of one of these two alternatives?

"Go to :menuselection:`Any application that use products --> Products --> Products` or :menuselection:`Any application that use products --> Products --> Product Variants`."

"Go to the Products menu (:menuselection:`Products --> Products` or :menuselection:`Products --> Products Variants`) from any application that uses products like Inventory or Sale"